src/cheats.c

```diff
@@ -214,25 +220,65 @@ static void cheat_10_minute_inspections()
 	window_invalidate_by_class(WC_RIDE);
 }
 
-static void cheat_increase_money(money32 amount)
+static void cheat_no_money(int enabled)
 {
-	money32 currentMoney;
+	if (enabled == 1) {
+		gParkFlags |= PARK_FLAGS_NO_MONEY;
```

Comment:
I would remove this for now. It can be done as a seperate PR where we sync the park parameter changes as those would also be useful to send over the network.